src/prebid.js

```diff
@@ -113,7 +113,12 @@ function resetPresetTargeting() {
   if (isGptPubadsDefined()) {
     window.googletag.pubads().getSlots().forEach(slot => {
       pbTargetingKeys.forEach(function(key){
-        slot.setTargeting(key,null);
+				// Reset Only Registerd dfp-adunits 
```

Comment:
Please use spaces rather than tabs in this section to match the rest of the file